components/dropdown/test/dropdown-menu.vdiff.js

```diff
@@ -45,6 +45,14 @@ const menuWithHeaderFooter = html`
 	<div slot="footer">Available 2020</div>
 `;
 
+const dropdownRadioMenu = html`
```

Comment:
If this is only used the once you can just put it inline with the rest of the dropdown